tools/profileBuilder/model/latest.go

```diff
@@ -31,11 +31,16 @@ func acceptAll(name string) bool {
 	return true
 }
 
-func ignorePreview(name string) (result bool) {
+var previewSubdir = fmt.Sprintf("%spreview%s", string(os.PathSeparator), string(os.PathSeparator))
+
+func ignorePreview(name string) bool {
+	// check if the path contains a /preview/ subdirectory
+	if strings.Contains(name, previewSubdir) {
+		return false
+	}
 	matches := packageName.FindStringSubmatch(name)
 	version := matches[3]
-	result = !strings.Contains(version, "-preview") && !strings.Contains(version, "-beta") // matches[2] is the `version` group
-	return
+	return !strings.Contains(version, "-preview") && !strings.Contains(version, "-beta") // matches[2] is the `version` group
```

Comment:
ok just to be sure, the policy is. if folder includes preview OR if its a preview api version we wont included. can you check please that the readme specifies this also for the latest profile

Reply:
The readme doesn't really explain the difference between the `latest` and `preview` profiles, will update.